features/list_news/lib/presentation/ui/list_news_page.dart

```diff
@@ -52,6 +52,8 @@ class ListNewsPage extends StatelessWidget {
   Widget _buildIos(BuildContext context) {
     return CupertinoPageScaffold(
       navigationBar: CupertinoNavigationBar(
+        heroTag: "list_new_page",
```

Comment:
You can delete `heroTag`.

Reply:
@putrautama007 Can you also add `transitionBetweenRoutes: false,` into other screens that implemented `CupertinoNavigationBar`?

Reply:
of courese